content/essentials/conditional-rendering.mdx

```diff
@@ -0,0 +1,182 @@
+  <Match when={condition1}>
+    <p>Outcome 1</p>
+  </Match>
+  <Match when={condition2}>
+    <p>Outcome 2</p>
+  </Match>
+</Switch>
+```
+
+Opting for this approach removes the need for nested `<Show>` components:
+
+```jsx
+// nested show
+<Show
+    when={x() > 10}
+      fallback={
+        <Show
+          when={5 > x()}
+          fallback={<p>{x()} is between 5 and 10</p>}
+        >
+          <p>{x()} is less than 5</p>
+        </Show>
+      }
+    >
+      <p>{x()} is greater than 10</p>
+    </Show>
+
+// switch/match alternative:
+
+<Switch fallback={<p>{x()} is between 5 and 10</p>}>
+  <Match when={x() > 10}>
+    <p>{x()} is greater than 10</p>
+  </Match>
+  <Match when={5 > x()}>
+    <p>{x()} is less than 5</p>
+  </Match>
+</Switch>
+```
+
+## Dynamic
+
+`<Dynamic>` introduces a dynamic component rendering capability, allowing you to select and render components based on varying data. 
+You can provide it with either a string representing a native HTML element or a component function. 
+Once you've specified the component to render, `<Dynamic>` will render the remaining props that you provide. 
+This flexibility makes it a versatile tool for adapting your UI based on changing data or user interactions:
+
+```jsx
+import { createSignal, Switch, Match, For } from "solid-js";
+
+const RedDiv = () => <div style="color: red">Red</div>;
+const GreenDiv = () => <div style="color: green">Green</div>;
+const BlueDiv = () => <div style="color: blue">Blue</div>;
+
+const options = {
+  red: RedDiv,
+  green: GreenDiv,
+  blue: BlueDiv
+}
+
+function App() {
+  const [selected, setSelected] = createSignal("red");
+
+  return (
+    <>
+      <select value={selected()} onInput={e => setSelected(e.currentTarget.value)}>
+        <For each={Object.keys(options)}>{
+          color => <option value={color}>{color}</option>
+        }</For>
+      </select>
+      <Dynamic component={options[selected()]} />
+    </>
+  );
+}
+```
+
+In this example, the `component` prop of `<Dynamic>` is assigned a value from the `options` array, which is indexed by the result of the `selected()` function call. 
+This dynamic assignment results in rendering the chosen component or element with the provided properties.
+
+Using `<Dynamic>` in scenarios like this can result in code that is more succinct and visually cleaner than alternative conditional rendering options. 
+If you were to replace it with a conditional rendering statement, the code would look something like this:
+
+```jsx
+<Switch fallback={<BlueDiv />}>
+  <Match when={selected() === 'red'}><RedDiv /></Match>
+  <Match when={selected() === 'green'}><GreenDiv /></Match>
+</Switch>
+```
+
+In this `<Switch>` statement, the `<Match>` components provide conditions to render specific components based on the result of `selected()`, which aligns with the behavior achieved using `<Dynamic>`. 
+However, `<Dynamic>` offers a more compact and readable way to achieve the same outcome.
+
+## Portal
```

Comment:
I'm kinda 50/50 on Portal([reference](https://discord.com/channels/722131463138705510/890648680053497896/1167825636904145036)), but great work on this section.